packages/mdc-chips/README.md

```diff
@@ -60,6 +60,21 @@ CSS Class | Description
 `mdc-chip__text` | Mandatory. Indicates the text content of the chip.
 `mdc-chip-set` | Mandatory. Indicates the set that the chip belongs to.
 
+### Sass Mixins
+
+To customize the colors of any part of the chip, use the following mixins. 
+
+Mixin | Description
+--- | ---
+`mdc-chip-corner-radius($radius)` | Customizes the corner radius for a chip
+`mdc-chip-accessible-color($fill-color)` | Customizes the background fill color for a chip, and updates the chip's ink and ripple color to meet accessibility standards
```

Comment:
Rename to `mdc-chip-fill-color-accessible` for consistency with other similar mixins in other packages (e.g. slider and toolbar). Then you can rename the parameter to just `$color` like the others here.